src/renderer/components/+namespaces/namespace-select.tsx

```diff
@@ -35,7 +33,7 @@ export class NamespaceSelect extends React.Component<Props> {
     ]);
   }
 
-  @computed get options(): SelectOption[] {
+  @computed.struct get options(): SelectOption[] {
```

Comment:
> In practice, computed.struct is less useful than it sounds. Only use it if changes in the underlying observables can still lead to the same output. For example, if we were rounding the coordinates first, the rounded coordinates might be equal to the previously rounded coordinates even though the underlying values aren't.

from here: https://mobx.js.org/computeds.html#computed-struct

Reply:
We have exact case when namespace store produces exactly same list on every context change. We shouldn't rerender then.